### ISSUE TITLE
Smoothing not working until pressure is toggled

### ISSUE THREAD

**Issue by notfood** [drawpile/Drawpile#954]
After commit 17081be8f7f9aea8037325692f86f19de16d060a was merged, smoothing appears to be disabled on program start. You're required to tinker with the input settings to initialize it.

https://user-images.githubusercontent.com/5370107/106167628-64715800-615b-11eb-8848-1d45c9bf43d4.mp4

**Comment by callaa** [comment]
Good catch. Looks like the bug only manifested when switching from the old version and there was no input preset yet assigned.
